packages/cli/tests/unit/fingerprint/fingerprintWatchCommand.spec.ts

```diff
@@ -114,10 +115,10 @@ describe(FingerprintWatchCommand, () => {
       it('send separate events for indexes that occur more than one second apart', async () => {
         placeMap();
         await once(handler, 'index');
-        jest.advanceTimersByTime(1024);
+        jest.advanceTimersByTime(MaxMSBetween * 1.1);
```

Comment:
1.1 here but 2 below looks inconsistent.  It should work with 1 too.

Reply:
MaxMSBetween is `1000`, so I didn't understand the choice of 1024, 2048, 512 etc since we are working with time here and not bytes.